### PR TITLE
vscode: update to 1.63.2.

### PR DESCRIPTION
**Pull request by shizonic**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

**Comment by paper42** [comment]
let's switch to vsed in do_configure if possible

**Comment by shizonic** [comment]
Switched to vsed where possible and bumped to 1.63.2.

File: srcpkgs/vscode/template [hunk 1]
```diff
@@ -33,7 +33,7 @@ do_configure() {
 	grep -rl --exclude-dir=.git -E $_TELEMETRY_URLS | xargs sed -i -E $_REPLACEMENT
 
 	# set used electron version
-	sed -i "s/^target .*/target \"${_electronver}\"/" .yarnrc
+	sed -e "s/^target .*/target \"${_electronver}\"/" -i .yarnrc
```
Comment (paper42) [inline srcpkgs/vscode/template:36]: **v**sed :)

Comment (shizonic) [reply srcpkgs/vscode/template]: Sorry, typo. :+1:

**Pull request merged**